src/language/collections.md

```diff
@@ -3,6 +3,9 @@ title: Collections
 description: Summary of the different types of collections in Dart.
 ---
 
+Lists, sets, and maps are all collections in Dart. You can learn more about each
```

Comment:
There are other collections in `dart:core` so perhaps we should be more specific. Something like:

```suggestion
Dart has built-in support for list, set, and map collections
and declaring them through literals.
You can learn more about each
```

Reply:
Could we say: "Dart has built in support for declaring list, set, and map collection literals", or does that change the meaning? 

The "...and declaring them..." clause reads a little awkward so I'm trying to consolidate them, but not sure I understand the distinction between supporting the collections vs. literals.

Reply:
I might be getting farther from the point here, but would it make sense to link to the library section on "collections", and the generics page on "literals"?

Reply:
I think it makes sense to link to the library tour on collections, but not as much generics on literals.

I do think supporting collections vs supporting literals are related but slightly different. Perhaps we can just say built-in support for collections, I think "built-in" implies syntactical support sufficiently.

```
Dart has built-in support for list, set, and map [collections][].
```

For the generic mention, maybe we can be more specific, since readers may not know what generics are for. Something like?

```
To learn more about configuring the types collections contain,
check out [Generics][].
```

Reply:
Thanks for iterating on this with me!